seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -64,3 +64,23 @@ avalanche_c,fraxswap,1,2022-09-05,0xc0d3943d0fb158b7fa111a20ae64e81044161e6a1cab
 avalanche_c,fraxswap,1,2022-06-16,0x17cc2def9d5869f604789dc0593af77b68fc6b51dd98e0dacf9f50d744d5d534,86,0xd24c2ad096400b6fbcd2ad8b24e7acbc21a1da64,1000.0,0x214db107654ff987ad859f34125307783fc8e387,228.50208340351497
 avalanche_c,fraxswap,1,2022-06-16,0xc7973b9adc9c15c4b536ac3682344420b8e8e541e801e0f451a52f685a10d002,65,0xd24c2ad096400b6fbcd2ad8b24e7acbc21a1da64,1000.0,0x214db107654ff987ad859f34125307783fc8e387,229.28068237184743
 avalanche_c,fraxswap,1,2022-09-05,0x608542c07700429cf096204f91a193341e40ae4059b7940d7eb5ad9f051836b8,4,0xd24c2ad096400b6fbcd2ad8b24e7acbc21a1da64,1052.0376768716717,0x214db107654ff987ad859f34125307783fc8e387,169.6623770097244
+avalanche_c,platypus_finance,1,2022-05-01,0xc9cf002b6422ef0a617284537683372c66b92e84e0b28fde8a0cc04a4eef519e,23,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,4994.308573,0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7,4994.787342
+avalanche_c,platypus_finance,1,2022-05-03,0x75dc4b71defb76d9888aabbd8771e8b38ee48fb41d43cc3ecae6fb73b3911c84,20,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,21009.805223,0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7,21012.319756
+avalanche_c,platypus_finance,1,2022-05-03,0x93bd89cf8a4d602d5cbc32446e5fb4bf9ed170f5ac72c2dc23294c8f5e1a8a05,26,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,1199719.208427,0xd586e7f844cea2f87f50152665bcbc2c279d8d70,1200000.0
```

Comment:
I think 3 trades would be enough for the test in this seed file.

Reply:
it doesn't hurt to have more, we can leave if passing